Wox.Core/Updater.cs

```diff
@@ -10,22 +10,25 @@
 using Squirrel;
 using Newtonsoft.Json;
 using Wox.Core.Resource;
+using Wox.Plugin.SharedCommands;
 using Wox.Infrastructure;
 using Wox.Infrastructure.Http;
 using Wox.Infrastructure.Logger;
+using System.IO;
 
 namespace Wox.Core
 {
     public class Updater
     {
         public string GitHubRepository { get; }
+        private bool IsPortableMode => Directory.Exists(Constant.PortableDataPath);
 
         public Updater(string gitHubRepository)
         {
             GitHubRepository = gitHubRepository;
         }
 
-        public async Task UpdateApp()
+        public async Task UpdateApp(bool silentIfLatestVersion = true)
         {
             UpdateManager m;
```

Comment:
I know this is not part of the commit, but can we maybe change m,u to better names to improve readbility -> manager etc...

Reply:
sure